src/components/Footer.js

```diff
@@ -1,56 +1,29 @@
-import "./Footer.css"
-// function Footer() {
-//     return (
-//     <div className="container">
-//         <div className="row" id="footer">
-//             <div className="col-md-12">
-//                 <small>Copyright &copy; Amanda 2022 </small>
-//             </div>
-//         </div>
-//     </div>
-//     )
-// }
- 
-// export default Footer
-
+import Logo from "../img/AmandaLogo.svg";
 import Container from 'react-bootstrap/Container';
 import Nav from 'react-bootstrap/Nav';
 import Navbar from 'react-bootstrap/Navbar';
-import Offcanvas from 'react-bootstrap/Offcanvas';
+import NavDropdown from 'react-bootstrap/NavDropdown';
 
-function Footer() {
+function BasicExample() {
   return (
-    <>
-      {['top'].map((expand) => (
-        <Navbar key={expand} bg="light" expand={expand} className="mb-3">
-          <Container fluid>
-            <Navbar.Brand href="#">Navbar Offcanvas</Navbar.Brand>
-            <Navbar.Toggle aria-controls={`offcanvasNavbar-expand-${expand}`} />
-            <Navbar.Offcanvas
-              id={`offcanvasNavbar-expand-${expand}`}
-              aria-labelledby={`offcanvasNavbarLabel-expand-${expand}`}
-              placement="end"
-            >
-              <Offcanvas.Header closeButton>
-                <Offcanvas.Title id={`offcanvasNavbarLabel-expand-${expand}`}>
-                  Offcanvas
-                </Offcanvas.Title>
-              </Offcanvas.Header>
-              <Offcanvas.Body>
-                <Nav className="justify-content-end flex-grow-1 pe-3">
-                  <Nav.Link href="#action1">Home</Nav.Link>
-                  <Nav.Link href="#action2">Skills</Nav.Link>
-                  <Nav.Link href="#action2">Experience</Nav.Link>
-                  <Nav.Link href="#action2">Projects</Nav.Link>
-                  <Nav.Link href="#action2">Contacts</Nav.Link>
-                </Nav>
-              </Offcanvas.Body>
-            </Navbar.Offcanvas>
-          </Container>
-        </Navbar>
-      ))}
-    </>
+    <Navbar bg="transparent" expand="lg">
+      <Container>
+        
+          <img src={Logo} alt="Logo" />
```

Comment:
Try to always include the width and height attributes when you use `<img>`. 
Reason: It makes your page perform better, because the browser doesn't have to compute the space needed.